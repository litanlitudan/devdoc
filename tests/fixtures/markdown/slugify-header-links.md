[[TOC]]

# HeadingOne
## Heading Two
### Heading-Three
#### Heading :checkered_flag: Four
##### Heading 🏁 Five
###### Heading ~`!@#$%^&*()_+[]{}\|;':",./<>? Six
##### This & That
