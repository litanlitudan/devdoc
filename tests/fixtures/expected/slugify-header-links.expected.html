<p><div class="table-of-contents"><ul><li><a href="#headingone">HeadingOne</a><ul><li><a href="#heading-two">Heading Two</a><ul><li><a href="#heading-three">Heading-Three</a><ul><li><a href="#heading-checkered_flag-four">Heading :checkered_flag: Four</a><ul><li><a href="#heading--five">Heading 🏁 Five</a><ul><li><a href="#heading-_-six">Heading ~`!@#$%^&*()_+[]{}\|;':",./<>? Six</a></li></ul></li><li><a href="#this--that">This & That</a></li></ul></li></ul></li></ul></li></ul></li></ul></div></p>
<h1 id="headingone">HeadingOne</h1>
<h2 id="heading-two">Heading Two</h2>
<h3 id="heading-three">Heading-Three</h3>
<h4 id="heading-checkered_flag-four">Heading 🏁 Four</h4>
<h5 id="heading--five">Heading 🏁 Five</h5>
<h6 id="heading-_-six">Heading ~`!@#$%^&amp;*()_+[]{}|;':&quot;,./&lt;&gt;? Six</h6>
<h5 id="this--that">This &amp; That</h5>
